packages/calling/src/CallHistory/types.ts

```diff
@@ -17,12 +16,28 @@ export type JanusResponseEvent = {
   message: string | null;
 };
 
+/**
+ * Interface for CallHistory Client
+ * This contains the APIs that helps to fetch the recent Call History Records.
+ */
 export interface ICallHistory extends Eventing<CallHistoryEventTypes> {
-  getSDKConnector: () => ISDKConnector;
-  getCallHistoryData: (
+  /**
+   * This API is used to fetch the the recent Call History Records.
```

Comment:
We could remove the word 'recent'. Call History already implies this.

Reply:
taken care